acceptance/tests/options/config_file/command_line_override.rb

```diff
@@ -1,9 +1,8 @@
-# This test is intended to demonstrate that all command-line flags take precedence
```

Comment:
Could you give this file a more specific name? I see other tests below whose names have been changed to also reference command line overriding, that include which setting they're testing.